airbyte-db/src/main/resources/schema.sql

```diff
@@ -60,8 +60,8 @@ CREATE
 CREATE
     TABLE
         ATTEMPTS(
-            id BIGSERIAL PRIMARY KEY,
-            job_id BIGSERIAL,
+            id BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
```

Comment:
what problem is this solving?

Reply:
https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_serial